packages/react/src/BackboneElementInput/BackbonElementInput.utils.ts

```diff
@@ -0,0 +1,56 @@
+import { InternalSchemaElement, InternalTypeSchema } from '@medplum/core';
+import React from 'react';
+
+export function splitRight(str: string, delim: string): [string, string] {
+  const lastIndex = str.lastIndexOf(delim);
```

Comment:
Does this need to check for `lastIndex === -1`?

Reply:
I will check that when I get around to writing tests!